db/db.js

```diff
@@ -0,0 +1,57 @@
+const mockLocationsData = require("./mockLocationsData.json");
+const mockHostelData = require("./mockHostelData.json");
+const mongoose = require("mongoose");
+mongoose.connect("mongodb://localhost/header");
+
+const db = mongoose.connection;
+db.on("error", console.error.bind(console, "db connection error:"));
+db.once("open", () => console.log("db connected successfully!"));
+
+const hostelSchema = mongoose.Schema({
+	id: Number,
+	location_id: Number,
+	photos: Array
+});
+
+const Hostel = mongoose.model("Hostel", hostelSchema);
+
+for (var j = 0; j < mockHostelData.length; j++) {
```

Comment:
var j = is a declaration + assignment. you can reuse var i unless you are using the two in conjunction. I feel this is more stylistic than anything, but you may want to take note of that if you didnt know.